pyfar/dsp/dsp.py

```diff
@@ -811,6 +813,315 @@ def regularized_spectrum_inversion(
     return inverse
 
 
+class interpolate_spectrum():
```

Comment:
I missed this the first time: Since this is a class, I'd use the class naming conventions here
```suggestion
class InterpolateSpectrum():
```

Reply:
thought about it as well and than used small letters in analogy to the scipy interpolation classes to indicate the function like use. But I don't have a strong opinion - maybe discuss with the others on Friday?

Reply:
Sure, I think the name `interp1d` is more a legacy kinda thing, to me it seems that the majority of scipy interpolators implemented as classes now follow the class naming convention (see [here](https://docs.scipy.org/doc/scipy/reference/interpolate.html)). There's some implemented as functions, not to be confused ;)

Reply:
True - I did not take a look at all interpolation classes before. Changed to `InterpolateSpectrum` :)